app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizardActions.js

```diff
@@ -32,6 +32,20 @@ export const planWizardExitedAction = () => dispatch => {
   });
 };
 
+export const planWizardNextAction = body => dispatch => {
+  dispatch({
+    type: PLAN_WIZARD_NEXT,
+    payload: body
+  });
+};
+
+export const planWizardBackAction = body => dispatch => {
+  dispatch({
+    type: PLAN_WIZARD_BACK,
+    payload: body
+  });
+};
+
```

Comment:
I don't see reducer cases for these actions.  Perhaps these were missed during cleanup?

Reply:
+1. What are we doing with these actions? Is the activeStepIndex from PlanWizard's react state being lifted up into Redux?

Reply:
I need a way for the Instance Properties step to tell us if we would be navigating `Back` or `Next`.

- Navigating `Back` -- possibly means an update in the VM list.
If the VM list is updated, we need to calculate the best fit flavors again in the Instance Properties step

- Navigating `Next` -- This takes us to the Advanced Options Step. 
If we navigate back to the Instance Properties step, we should not be calculating the best fit flavors again because we might be overriding the flavor/security group values that the user may have edited previously.

So that's really the premise for these new actions in PlanWizardActions

I don't need the Reducer in PlanWizard, since I don't need to set any states there.
Instead I need to handle the  `PLAN_WIZARD_NEXT` case in `PlanWizardInstancePropertiesStepReducer`, where I am setting a flag that tells me if we need to recalculate the best fit flavors depending on where we are in the Wizard.
(recalculate only if in VM Step)

Reply:
ahh, ok, I'm with ya now.

Maybe, can we rename these actions though?  Currently, they read as if they should control whether or not the Wizard advances forward or backward.

Maybe something like `setPlanWizardForwardMetadata` (ugh) or `setPlanWizardBackAdditionalState` (uggghhh) or something

Naming is not fun

Reply:
Sounds good... will figure something out.

Heads-up - I may actually end up using either the ugh or the uggghhh names... :)  -- we will see.

Reply:
I went with `setMetadataWithNextButtonClickedAction` (and `setMetadataWithBackButtonClickedAction`) -- it's a mouthful, but hopefully there are no ambiguities now.